kbase-extension/static/kbase/js/widgets/function_output/kbaseReportView.js

```diff
@@ -913,9 +907,11 @@ define([
             this.loading(false);
             let errString = 'Unknown error.';
             console.error(error);
-            if (typeof error === 'string') errString = error;
-            else if (error.error && error.error.message) errString = error.error.message;
-            else if (error.error && error.error.error && typeof error.error.error === 'string') {
+            if (typeof error === 'string') {
+                errString = error;
+            } else if (error.error && error.error.message) {
+                errString = error.error.message;
+            } else if (error.error && error.error.error && typeof error.error.error === 'string') {
```

Comment:
Do you also need to check for `error.error.error.error`? ;)

Reply:
Sigh. Yes. For now.